src/google/protobuf/compiler/java/java_message_field.cc

```diff
@@ -299,6 +300,7 @@ GenerateBuilderMembers(io::Printer* printer) const {
     "$name$Builder_.setMessage(value);\n",
 
     "$set_has_field_bit_builder$\n"
+    "// @@protoc_insertion_point(builder_field_modifier_scope_after:$containing_type_full_name$.set$capitalized_name$)\n"
```

Comment:
Yes, please. @@protoc_insertion_point(accessor_name:$full_name$) is the way we name these insertion points and should be consistent through out.